software/contrib/quantizer.py

```diff
@@ -449,10 +451,10 @@ def read_quantize_output(self):
         self.input_voltage = ain.read_voltage(500)   # increase the number of samples to help reduce noise
         self.quantize(self.input_voltage)
 
-        cv1.voltage(self.output_voltage)
+        cv1.voltage(min(self.output_voltage,self.max_voltage))
```

Comment:
[discussion] I feel like this change probably introduces a bug in the quantizer (or maybe points out an existing one?) If the quantized voltage is above the max voltage, the output shouldn't be the max voltage, it should be the next lower step in the scale. RIght?

Reply:
I concur; this would definitely introduce a bug. The better fix would be to modify `experimental/quantizer.py` to respect the voltage range. I can take a look at that tomorrow.

Reply:
I've reverted this change and modified the experimental Quantizer class to drop down in octaves if it outputs something that's outside the valid voltage range. This isn't necessarily ideal, but it's better than clamping to the max voltage, and it seems unlikely to ever be an issue under most use cases; I don't see people sending 10V inputs into a quantizer very often.

If this isn't acceptable, I can revise the math to round to the highest on-scale note instead.

Reply:
The 'highest on-scale note' makes more sense to me as what the user would expect, but I defer to you on the behavior of the quantizer that you designed. I think it's fine like this, and we can address it again if it causes confustion.

Reply:
I think the amount of work for a highest-on-scale calculation would fall outside the scope of this MR. I say we move ahead with things as they stand and I'll get to work on fixing this edge case in the quantizer separately.

Reply:
The changes to the quantizer are done. Probably out of scope for this MR, but they're ready for a review.